### ISSUE TITLE
`rultor merge` fails due to `proguard` obfuscation activation on JDK 21

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/lints#1305]
Every `@rultor merge` fails ([example](https://github.com/objectionary/lints/pull/1235#issuecomment-5392250073)):

```
Failed to execute goal com.github.wvengen:proguard-maven-plugin:2.7.0:proguard (default)
on project lints: Obfuscation failed (result=1)
```

**Why.** The `proguard` profile activates on `<jdk>[,22)</jdk>` (`pom.xml:565-568`). Rultor runs `yegor256/java` (JDK 21), so the profile is active there. Locally we're on JDK 23, so it never activates -> `mvn clean install -Pqulice` passes for us and the failure only shows up in rultor.

ProGuard 2.7.0 on JDK 21 needs `--add-opens` to read JDK classes. The **release** script already knows this (`.rultor.yml:28-29`) - it exports `MAVEN_OPTS` *and* passes `!proguard`. The **merge** script (`.rultor.yml:20`) does neither:

```yaml
merge:
  script: |
    mvn clean install -ntp -P!hone,qulice --errors -Dstyle.color=never
```

**Fix.** Align merge with release:

```yaml
merge:
  script: |
    export MAVEN_OPTS="--add-opens java.base/java.util=ALL-UNNAMED --add-opens java.base/java.lang.reflect=ALL-UNNAMED"
    mvn clean install -ntp -P!hone,!proguard,qulice --errors -Dstyle.color=never
```

Obfuscation adds nothing to a merge check anyway - it only matters for the deployed artifact, and release already skips it.

Analysis originally posted in #1238; filing separately so it doesn't get lost.

**Comment by 0crat** [comment]
@volodya-lombrozo Thanks for reporting a new bug! You've earned +12 points. By reporting bugs, you help our project improve its quality. If you find anything else in the repository that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) as good as you might expect, [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. Your running score is [+401](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).

**Comment by volodya-lombrozo** [comment]
Fixed in the #1306

**Comment by 0crat** [comment]
@yegor256 Thanks for closing this issue! You've earned +12 points for this: +8 as a basis; +4 for resolving it in less than 12 hours. Your running score is [+1876](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
